src/test/kotlin/build/buf/gradle/AbstractBreakingTest.kt

```diff
@@ -62,7 +62,7 @@ abstract class AbstractBreakingTest : AbstractBufIntegrationTest() {
         assertThat(result.output).contains("Cannot configure $BUF_BREAKING_TASK_NAME against latest release and a previous version.")
     }
 
-    private fun checkBreaking() {
+    protected fun checkBreaking() {
```

Comment:
Do these need to be protected now? I saw a few commits go by with them flip flopping?

Reply:
They can be private, not sure why my changes were not holding. Should be fixed now.